alerts/jira.go

```diff
@@ -144,16 +144,14 @@ func getSummary(ctx AlertContext) string {
 	switch {
 	case ctx.Task.Details.TimedOut:
 		subj.WriteString("Timed Out: ")
-	case len(failed) == 1:
-		subj.WriteString("Failure: ")
-	case len(failed) > 1:
-		subj.WriteString("Failures: ")
-	case ctx.Task.Details.Description == task.AgentHeartbeat:
```

Comment:
Note that this is only ever the case if `ctx.Task.Details.TimedOut` is true, so we never hit this branch, so I removed it.